physx/src/heightfield.rs

```diff
@@ -8,12 +8,10 @@
 /*!
 
 */
-use super::{
-    cooking::{Cooking, HEIGHT_SCALE, XZ_SCALE},
-    geometry::Geometry,
-};
+use super::{cooking::Cooking, geometry::Geometry};
 use enumflags2::BitFlags;
-use glam::Vec3;
+
+use glam::f32::Vec3;
```

Comment:
@Chris-Nicholls Is this related to the other glam::Vec3/PxVec3 change you requested for the TriMesh Desc?

Reply:
No, this shouldn't be here. glam::Vec3 is the same as glam::f32::Vec3

Reply:
```suggestion
use glam::Vec3;
```

Reply:
```suggestion
use glam::Vec3;
```

Reply:
Great, thanks for the fix.